crates/bevy_scene/src/dynamic_scene.rs

```diff
@@ -26,7 +26,9 @@ use serde::Serialize;
 /// [`GlobalTransform`](bevy_transform::components::GlobalTransform) components)
 #[derive(Asset, TypePath, Default)]
 pub struct DynamicScene {
+    /// Resources used by the dynamic scene.
```

Comment:
resources are not used by the scene but contained

Reply:
```suggestion
    /// Resources stored in the dynamic scene.
```